tests/xtriggers/00-wall_clock.t

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2017 NIWA
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Test clock xtriggers
+. $(dirname $0)/test_header
+
+run_suite() {
+  cylc run --debug $1 -s START=$2 -s HOUR=$3 -s OFFSET=$4
```

Comment:
Need `--no-detach` these days.

Reply:
Oops, I guess I forgot to check test battery results and clean up after last rebase to master! (it was all passing before that - sans the Kafka problem above).

Reply:
done